### PR TITLE
Better jamulus.desktop installation

### PR DESCRIPTION
**Pull request by atsampson**
Two suggestions from @jtojnar in the #331 comments (thanks!): use QMAKE_SUBSTITUTES to expand the binary name, and install the icon as well.

I've used the large version of the icon as distributions/debian/rules does, rather than the small one that AUR installs.

I note the .desktop file only has a French translation at the moment, so if any other language translators are reading this...

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,4 @@ distributions/opus*
 distributions/jack2
 distributions/claudio_piano.sf2
 distributions/fluidsynth*
+distributions/jamulus.desktop
```
Comment (corrados) [inline .gitignore:34]: Why did you add the jamulus.desktop to the git ignore file?

Comment (corrados) [reply .gitignore:34]: Oh, I see. You generate this file now. Please ignore my above comment...

File: distributions/jamulus.desktop.in [hunk 2]
```diff
@@ -4,7 +4,7 @@ Comment=Jam Session
 Comment[fr]=Séance de bœuf
 GenericName=Internet Jam Session Software
 GenericName[fr]=Logiciel de séance de bœuf sur Internet
-Exec=jamulus
+Exec=$$TARGET
```
Comment (corrados) [inline distributions/jamulus.desktop.in:7]: You should add the new file jamulus.desktop.in to the DISTFILES in the .pro file.

File: Jamulus.pro [hunk 3]
```diff
@@ -308,11 +308,17 @@ win32 {
     }
     APPSDIR = $$absolute_path($$APPSDIR, $$PREFIX)
     desktop.path = $$APPSDIR
+    QMAKE_SUBSTITUTES += distributions/jamulus.desktop.in
     desktop.files = distributions/jamulus.desktop
-    # the .desktop file assumes the binary is called jamulus
-    contains(CONFIG, "noupcasename") {
-        INSTALLS += desktop
+    INSTALLS += desktop
```
Comment (corrados) [inline Jamulus.pro:313]: Why not add the desktop and icons in one step?

Comment (atsampson) [reply Jamulus.pro]: OK!

File: distributions/jamulus.desktop.in (hunk 2 above)
Comment (atsampson) [reply distributions/jamulus.desktop.in:7]: Done - and the same for the .png.

**Comment by corrados** [comment]
Thanks.

**Comment by jtojnar** [comment]
Thank you. cc @seb314
